docs/earthpy-data-subsets.rst

```diff
@@ -0,0 +1,83 @@
+=====================
+EarthPy Data Subsets
+=====================
+
+EarthPy comes with the ability to download example datasets using
+``earthpy.data.get_data()``. These datasets are used in example documentation
+on how to use EarthPy, as well as in the EarthPy related lessons on
+earthdatascience.org. There are many keys that can be used to get data
+this way, all of which can be found below.
+
+Data Keys
+=========
+
+co-flood-extras
+---------------
+
+Contains two additional csv files for the Colorado Flood exercise on earthdatascience.org.
+Includes precipitation and `temperature <https://figshare.com/articles/Week_2_earth_analytics/4502138>`_
+data for the Boulder flood in 2013.
+
+colorado-flood
+--------------
+
+Main `dataset for the Colorado Flood <https://figshare.com/articles/Earth_Analytics_Python_Colorado_Flood_Teaching_Subset/6815945>`_
+exercise on https://www.earthdatascience.org. Contains precipitation,
+temperature, lidar, and clipping data for the 2013 Boulder Floods.
+
+spatial-vector-lidar
+--------------------
+
+Main `dataset for vector and lidar <https://figshare.com/articles/Week_4_Earth_Analytics_teaching_data_subset_-_lidar_and_insitu_for_California/4620268>`_
+lessons on https://www.earthdatascience.org. Contains shapefiles for
+a variety of exercises, including road, county, and country shapefiles. Also contains
+lidar data from NEON for research done in California, as well as supporting data for the lidar data.
+
+cold-springs-modis-h5
+---------------------
+
+`Modis satellite data <https://figshare.com/articles/Earth_Analytics_Cold_Springs_Fire_Remote_Sensing_Data/6083210>`_
+for the 2016 Cold Springs fire in h5 format.
+
+cold-springs-fire
+-----------------
+
+`Satellite and vector data for the 2016 Cold Springs fire <https://figshare.com/articles/Earth_Analytics_Cold_Springs_Fire_Remote_Sensing_Data/6083210>`_.
+Includes Landsat, Modis, and NAIP data.
+
+cs-test-naip
+------------
+
+Naip imagery related to the 2016 Cold Springs fire.
+
+cs-test-landsat
+---------------
+
+Landsat imagery related to the 2016 Cold Springs fire.
+
+ndvi-automation
+---------------
+
+`Landsat imagery and vector files <https://figshare.com/articles/Earth_Analytics_Python_HOMEWORK_Landsat_8_2017_for_SJER_and_HARV_Download_from_Earth_Explorer/7272500>`_
+for the automation assigment on https://www.earthdatascience.org. The data covers sites studied by NEON.
```

Comment:
Be more specific about the sites. Spell them out. Example: San Joaquin Experimental Range (SJER) in California.